aladdin-container.sh

```diff
@@ -132,9 +131,17 @@ function _initialize_helm() {
     fi
 }
 
-function add_and_set_authentication_users() {
-    # If authentication is enabled, add the appropriate users to our kubeconfig
+function _handle_authentication_config() {
+    # This function adds appropriate users to kubeconfig, and exports necessary AUTHENTICATION variables
+    # export AUTHENTICATION_ENABLED for change-permissions and bash command
+    export AUTHENTICATION_ENABLED="$(_extract_cluster_config_value authentication_enabled)"
```

Comment:
these variables were previous extracted in aladdin.sh and then passed into the docker run command. now they're just loaded at use time, and exported if required